### DISCUSSION TITLE
Multiple instances of birdseye

### DISCUSSION THREAD

**Discussion by Deishelon** [blakeblackshear/frigate#7176] [Ideas]
Will it be possible to allow multiple modes of birdseye? 

For example, in my use-case I want to have one display to only display birdseye in `objects` mode, where another display will show `continuous` always.

**Comment by NickM-27** [comment]
There is a pinned feature request for this https://github.com/blakeblackshear/frigate/issues/6531

**Reply by Deishelon** [reply]
I guess I can add my use case for multiple modes in that issue, but that issue specifically talks about camera groups and not the operating mode

**Reply by NickM-27** [reply]
The request is for multiple birdseye views, the user requesting doesn't decide the implementation
